x-pack/plugins/security_solution/cypress/objects/case.ts

```diff
@@ -44,3 +44,189 @@ export const serviceNowConnector: Connector = {
 };
 
 export const TIMELINE_CASE_ID = '68248e00-f689-11ea-9ab2-59238b522856';
+export const ids = {
+  jira: '000e5f86-08b0-4882-adfd-6df981d45c1b',
+  sn: '93a69ba3-3c31-4b4c-bf86-cc79a090f437',
+  resilient: 'a6a8dd7f-7e88-48fe-9b9f-70b668da8cbc',
+};
+
+export const mockConnectorsResponse = [
+  {
+    id: ids.jira,
+    actionTypeId: '.jira',
+    name: 'Jira',
+    config: {
+      incidentConfiguration: {
+        mapping: [
+          { source: 'title', target: 'summary', actionType: 'overwrite' },
+          { source: 'description', target: 'description', actionType: 'overwrite' },
+          { source: 'comments', target: 'comments', actionType: 'append' },
+        ],
+      },
+      isCaseOwned: true,
+      apiUrl: 'https://siem-kibana.atlassian.net',
+      projectKey: 'RJ',
+    },
+    isPreconfigured: false,
+    referencedByCount: 0,
+  },
+  {
+    id: ids.resilient,
+    actionTypeId: '.resilient',
+    name: 'Resilient',
+    config: {
+      incidentConfiguration: {
+        mapping: [
+          { source: 'title', target: 'name', actionType: 'overwrite' },
+          { source: 'description', target: 'description', actionType: 'overwrite' },
+          { source: 'comments', target: 'comments', actionType: 'append' },
+        ],
+      },
+      isCaseOwned: true,
+      apiUrl: 'https://ibm-resilient.siem.estc.dev',
+      orgId: '201',
+    },
+    isPreconfigured: false,
+    referencedByCount: 0,
+  },
+  {
+    id: ids.sn,
+    actionTypeId: '.servicenow',
+    name: 'ServiceNow',
+    config: {
+      incidentConfiguration: {
+        mapping: [
+          { source: 'title', target: 'short_description', actionType: 'overwrite' },
+          { source: 'description', target: 'description', actionType: 'overwrite' },
+          { source: 'comments', target: 'comments', actionType: 'append' },
+        ],
+      },
+      isCaseOwned: true,
+      apiUrl: 'https://dev65287.service-now.com',
+    },
+    isPreconfigured: false,
+    referencedByCount: 0,
+  },
+];
+export const responses = {
```

Comment:
nit -> What do you think about renaming it to `externalServiceResponses` or something similar?